e3tools/local-development-portal
# Add filters and clean code

# How to test?

- [ ] Create your administrative_levelstable in CouchDB
- [ ] Create your  village_attachments table in CouchDB with same adm_id
- [ ] In village detail,
- Check that when no images in the attachment table, then you would see a placeholder
- Check that when images in the attachment table, then you would see a carousel
- [ ] Attachments list 
- Navigate to attachment list by clicking on the image carousel in the village detail
- Check that when no images in the attachment table, then you would see a message of "No attachments"
- Check that when images in the attachment table, then you would see a list of attachments.
- [ ] Attachment modal
- Click on a single attachment and a modal would open with the carousel of the attachments
- Click on the image or document of the modal and check that the file opens in another tab
- [ ] Attachment filters
- Check that the filters load the tasks, activities and phases of the adm_id 
- Check when filtering that results show the correct list
- Check when you reload page, the filters load the initial values that are in the query params
- Check when no result found, the message is displayed in the list and that you can see filter options
